### DISCUSSION TITLE
Барахолка

### DISCUSSION THREAD

**Discussion by GRbit** [b4ck5p4c3/0x08.in#370] [Ideas] [answered]
Есть мысль организовать барахолку.

Большая часть из нас любит и уважает всякие хорошо сделанные технические вещи. Уверен у каждого со временем накапливается куча вещей которые с одной стороны "хорошая же штука, ценная", с другой стороны никакого применения в быту не находят годами. Либо это какая-то модифицированная вещь, которая для обывателя "ой фу, вы разбирали, вдруг она сломается теперь?", а для своих это может быть реально нужный  важный мод. Напимер встроить в какой-нибудь прибор питаемый от 9v порт type c для подключения к павербанку.

Мне было бы приятно расставаться с такими вещами зная что они окажутся в руках достойного человека. Надеюсь в этом плане надйутся единомышленники, которые как и я готовы расстаться с вещью подешевле, рассказать про какие-то доработки/допилы, нюансы изготовления и другие проблемы, при условии что вещь пойдёт к человеку с руками от плечей. Да и просто приятно помочь брату технократу с приобретением нужной ему вундервафли)

Основные поинты барахолки:
1. Администрация ответственности ни за что не несёт.
2. Подразумевается, что участник предлагающий что-то сообществу делает это на эксклюзивных условиях. Например на авито у него объявление о продаже старого ноута висит за 20к, а кому-то из сообщества готов уступить за 10к.
3. Можно и нужно предлагать модифицированные вещи со своими костылями дополняющими/исправляющими оригинальную конструкцию.
4. Предприятие некомерческое, так что 

Следить за этим на мой взгляд бесполезно, всегда найдётся участник который в первую очередь будет желать заработка, так что вопросы саморегулирования предлагаю отдать на откуп сообществу.

Ещё мысль: некоторые лоты по желанию участников можно продавать в формате аукциона. Владелец говорит "хочу с этого 1к рублей". Хакспейс промоутит аукцион, а участники в течении скажем недели могут повышать ставки. Всё что выше 1к идёт на аренду хакспейса)

**Comment by heeeyflo** [comment, marked as answer]
Спасибо что органично поделился своей идеей, это здорово!
Буквально за пару часов до тебя, другой участник уже создал дискуссию по этому вопросу: #368. Сможешь перенести туда свой пост, чтобы все обсуждение было в одном месте?

**Reply by GRbit** [reply]
лол, я вовремя)

**Reply by GRbit** [reply]
такс, и как теперь дискашн закрыть?

**Reply by heeeyflo** [reply]
Ща сделаем :)
